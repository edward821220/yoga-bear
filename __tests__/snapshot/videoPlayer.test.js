import { create } from "react-test-renderer";
import { ThemeProvider } from "styled-components";
import VideoPlayer from "../../src/components/videoPlayer";

const theme = {
  colors: {
    color1: "#fff",
    color2: "#654116",
    color3: "#5d7262",
    color4: "#f7ecde",
    color5: "#f2deba",
    color6: "#00000080",
    color7: "#3f3f3f",
    color8: "#f4f7f7",
  },
};
/* eslint-disable */
describe("Jest Snapshot testing suite", () => {
  test("Matches DOM Snapshot", () => {
    const tree = create(
      <ThemeProvider theme={theme}>
        <VideoPlayer />
      </ThemeProvider>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
