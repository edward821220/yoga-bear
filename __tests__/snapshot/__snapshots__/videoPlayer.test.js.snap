// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches DOM Snapshot 1`] = `
<div
  className="fullscreen"
>
  <div
    className="sc-bcXHqe jpzefr"
    onKeyDown={[Function]}
    onMouseMove={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  >
    <video
      autoPlay={true}
      className="sc-gswNZR jLZYSS"
      data-testid="video"
      onClick={[Function]}
      onEnded={[Function]}
      onTimeUpdate={[Function]}
    />
  </div>
  <video
    className="sc-hBxehG hVIPgj"
    crossOrigin="anonymous"
  />
</div>
`;
